index.bs

```diff
@@ -1674,23 +1673,6 @@ partial interface MLGraphBuilder {
     1. Return |output|.
 </details>
 
-#### {{MLGraphBuilder/clamp(options)}} #### {#api-mlgraphbuilder-clamp-options}
-<div>
-    **Arguments:**
-        - *options*: an optional {{MLClampOptions}}. The optional parameters of the operation.
-    **Returns:**
-        - an {{MLActivation}}. The operator representing the clamp operation.
-</div>
-
-<details open algorithm>
-  <summary>
-    The <dfn method for=MLGraphBuilder>clamp(|options|)</dfn> method steps are:
-  </summary>
-    1. If [=checking clamp options=] given |options| returns false, then [=exception/throw=] a {{TypeError}}.
```

Comment:
Inline the "check clamp options" algorithm now that there's only one invocation?